signup.css

```diff
@@ -0,0 +1,106 @@
+*{
+  margin:0;
+  padding:0;
+  box-sizing: border-box;
+}
+body{
+  background-color: #f7f4f4;
+  font-family:sans-serif;
+}
+.blue{
+  color:#1F59BB;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#1F59BB" to be "#1f59bb" (color-hex-case)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5576441)